WordPress/Classes/ViewRelated/Site Creation/FinalAssembly/SiteCreationRotatingMessageView.swift

```diff
@@ -0,0 +1,189 @@
+import UIKit
+
+class SiteCreationRotatingMessageView: UIView {
+    private struct Margins {
+        static let horizontalMargin: CGFloat = 0
+        static let verticalMargin: CGFloat = 0
+
+        static let horizontalSpacing: CGFloat = 15
+    }
+
+    private struct Parameters {
+        //Animation parameters
+        static let transitionAnimationDuration = TimeInterval(0.5)
+        private static let statusDisplayDuration = TimeInterval(2)
+
+        //Adjust the total display time to take the fade into consideration
+        static var totalStatusDisplayDuration: TimeInterval {
+            return statusDisplayDuration + transitionAnimationDuration
+        }
+    }
+
+    /// An array of status messages to rotating through
+    private(set) var statusMessages: [String]!
```

Comment:
Since this is a property we're setting on the constructor we can remove the `!`
```suggestion
    private(set) var statusMessages: [String]
```